transmart-data/updatedb/release-17.1-HYVE-6.0/postgres/visit_dimension_pk.sql

```diff
@@ -0,0 +1,27 @@
+-- Change type of encounter_num to serial
+alter table i2b2demodata.visit_dimension
+  column encounter_num type serial;
+
+alter table i2b2demodata.observation_fact
+  column encounter_num type integer;
```

Comment:
```suggestion
  alter column encounter_num type integer;
```

Reply:
The migration script for postgres should now be fixed.